.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,78 @@
+name: Release Please
+on:
+  push:
+    branches:
+      - develop
+
+permissions:
+  actions: read # Read the metrics
+  contents: write # to be able to publish a GitHub release
+  issues: write # to be able to comment on released issues
+  pull-requests: write # to be able to comment on released pull requests
+  id-token: write # to enable use of OIDC for npm provenance / AWS
+
+env:
+  MISE_PYTHON_COMPILE: false
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    outputs:
+      cli-release-created: ${{ steps.release.outputs['.--release_created'] }}
+      server-release-created: ${{ steps.release.outputs['gitops_server--release_created'] }}
+      helm-release-created: ${{ steps.release.outputs['charts/gitops--release_created'] }}
+      cli-version: ${{ steps.release.outputs['.--tag_name'] }}
+      server-version: ${{ steps.release.outputs['gitops_server--tag_name'] }}
+      helm-version: ${{ steps.release.outputs['charts/gitops--tag_name'] }}
+      sha: ${{ steps.release.outputs.sha }}
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        id: release
+        with: {}
+  publish_helm_chart:
+    runs-on: ubuntu-latest
+    needs: release-please
+    steps:
+      - uses: actions/checkout@v3
+      - name: Publish Helm charts
+        uses: stefanprodan/helm-gh-pages@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          app_version: "${{ needs.release-please.outputs.server-version }}"
+          chart_version: "${{ needs.release-please.outputs.helm-version }}"
+  publish_to_pypi:
```

Comment:
Do we need conditionals on these publish etc only when the output has been created?

I'll let you merge and go ahead ( so that it can be tested) but i suspect you will need to.

Reply:
makes sense, I've added in the conditions 👍🏼